package.json

```diff
@@ -8,7 +8,8 @@
     "lint": "tslint \"src/**/*.ts\"",
     "test": "ng test",
     "pree2e": "webdriver-manager update",
-    "e2e": "protractor"
+    "e2e": "protractor",
```

Comment:
@isuruAb Actually, these changes came when I tried to deploy the branch. My cloned repo was not updated. Sorry for that.